guides/coding_style/c++/README.md

```diff
@@ -5,28 +5,58 @@
 > All demonstrates are VALID version
 
 ## Index
-- [Code width and alignment](#code-width-and-alignment)
-- [Including Header Files](#including-header-files)
 - [Comments](#comments)
-- [Namespaces](#namespaces)
+- [Code width and alignment](#code-width-and-alignment)
+- [Spaces](#spaces)
+- [Indentations](#indentations)
+- [Newlines](#newlines)
+- [Braces](#braces)
 - [Naming](#naming)
   <details open>
     <summary>categories</summary>
     <ul>
-        <li><a href="#file-names">File Names</a></li>
-        <li><a href="#type-names">Type Names</a></li>
-        <li><a href="#variable-names">Variable Names</a></li>
-        <li><a href="#function-names">Function Names</a></li>
-        <li><a href="#macro-names">Macro Names</a></li>
-        <li><a href="#exceptions-to-naming-rules">Exceptions to Naming Rules</a></li>
+        <li><a href="#file-names">File names</a></li>
```

Comment:
I think the previous link names were better, since they were titles, and the capital letters made sense, although this is a matter of preference.